semantic-conventions/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ Please update the changelog as part of any significant pull request.
 
 ## Unreleased
 
-- ...
+- Add definitions for metrics in YAML semconv, plus generation to markdown
```

Comment:
You don't add defintions for metrics, you add support for them. Maybe copy the wording from the changelog entry below which added the "scope" type.

Reply:
Done